tests/CustomElements/v1/js/upgrade.js

```diff
@@ -61,4 +61,32 @@ suite('Upgrades', function() {
     assert.instanceOf(e2, X2);
   });
 
+  suite('customised build-in elements', function() {
```

Comment:
It would be good to also test what happens if an invalid `{ is }` is passed.

Some other questions, do you think it would be prudent to add a test for every single possible built-in that can be extended here?